activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1108,22 +1108,22 @@ def foreign_keys(table_name)
       #   ALTER TABLE "articles" ADD CONSTRAINT fk_rails_e74ce85cbc FOREIGN KEY ("author_id") REFERENCES "authors" ("id") ON DELETE CASCADE
       #
       # The +options+ hash can include the following keys:
-      # [<tt>:column</tt>]
+      # [+:column+]
       #   The foreign key column name on +from_table+. Defaults to <tt>to_table.singularize + "_id"</tt>
-      # [<tt>:primary_key</tt>]
+      # [+:primary_key+]
       #   The primary key column name on +to_table+. Defaults to +id+.
-      # [<tt>:name</tt>]
+      # [+:name+]
       #   The constraint name. Defaults to <tt>fk_rails_<identifier></tt>.
-      # [<tt>:on_delete</tt>]
+      # [+:on_delete+]
       #   Action that happens <tt>ON DELETE</tt>. Valid values are +:nullify+, +:cascade+, and +:restrict+
-      # [<tt>:on_update</tt>]
+      # [+:on_update+]
       #   Action that happens <tt>ON UPDATE</tt>. Valid values are +:nullify+, +:cascade+, and +:restrict+
```

Comment:
```suggestion
      #   Action that happens +ON UPDATE+. Valid values are +:nullify+, +:cascade+, and +:restrict+
```

Reply:
For whatever reason, spaces break up the short-form fixed stuff:

```
echo "+ON UPDATE+" | rdoc --pipe

<p>+ON UPDATE+</p>
```